event/addrs.go

```diff
@@ -55,6 +55,11 @@ type UpdatedAddress struct {
 // If the event producer is not capable or producing diffs, the Diffs field will
 // be false, the Removed list will always be empty, and the Action for each
 // UpdatedAddress in the Current list will be Unknown.
+//
+// In addition to the above, EvtLocalAddressesUpdated also contains the updated peer.PeerRecord
+// for the Current set of listen addresses, wrapped in a record.Envelope and signed by the Host's private key.
+// This record can be shared with other peers to inform them of our listen addresses in
```

Comment:
Listen address != diallable addresses.

```suggestion
// This record can be shared with other peers to inform them of what we believe are our  diallable addresses
```